swdc

```diff
@@ -66,13 +66,40 @@ function applyFixture()
   docker-compose run --rm cli mysql -h mysql -u root -proot $SHOPWARE_PROJECT < "${DIR}/fixtures/${FIXTURE_NAME}.sql"
 }
 
+function isComposerProject()
+{
+    if [ -d "$SHOPWARE_FOLDER/app" ]; then
+        echo "true"
+    fi
+}
+
+function buildComposerProjectEnvFile()
+{
+  echo "SHOPWARE_VERSION_TEXT=\"$SHOPWARE_PROJECT (local docker)\"" >> "$SHOPWARE_FOLDER/.env"
+  echo "SHOPWARE_REVISION=\"`git rev-parse HEAD`\"" >> "$SHOPWARE_FOLDER/.env"
+  echo "DATABASE_URL=\"mysql://root:root@mysql:3306/$SHOPWARE_PROJECT\"" >> "$SHOPWARE_FOLDER/.env"
+  echo 'ADMIN_EMAIL="demo@demo.com"' >> "$SHOPWARE_FOLDER/.env"
+  echo 'ADMIN_NAME="Don Joe"' >> "$SHOPWARE_FOLDER/.env"
+  echo 'ADMIN_USERNAME="demo"' >> "$SHOPWARE_FOLDER/.env"
+  echo 'ADMIN_PASSWORD="demo"' >> "$SHOPWARE_FOLDER/.env"
+  echo "SHOP_URL=\"http://$SHOPWARE_PROJECT.dev.localhost\"" >> "$SHOPWARE_FOLDER/.env"
+  echo 'IMPORT_DEMODATA=y' >> "$SHOPWARE_FOLDER/.env"
+}
+
 case "$1" in
   build)
     checkParameter
     clearCache
-    docker-compose run --rm -eANT_OPTS=-D"file.encoding=UTF-8" cli ant -Dapp.host=$SHOPWARE_PROJECT.dev.localhost -Ddb.host=mysql -Ddb.user=root -Ddb.password=root -Ddb.name=$SHOPWARE_PROJECT -f /var/www/html/$SHOPWARE_PROJECT/build/build.xml build-unit
-    fixHooks
-    fixDeveloperConfig
+    if [ `isComposerProject` ]; then
+      buildComposerProjectEnvFile
```

Comment:
Move this line after composer install.
Cause shopware thinks its a update and throws a exception :)

Reply:
I saw this as well but ignored it :see_no_evil: I will rearrange that